modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-new-submissions/index.css

```diff
@@ -0,0 +1,55 @@
+.applications-new-submissions-container {
+	min-height: 267px;
+}
+
+.applications-new-submissions-container
+	.applications-new-submissions-header
+	.applications-new-submissions-title::after {
+	background-color: #4d85ff;
+	content: '';
+	height: 1px;
+	left: 28px;
+	min-width: 20px;
+	position: absolute;
+	top: 50px;
+}
+
+.applications-new-submissions-container .applications-new-submissions-select {
+	border-color: var(--white);
+	padding: 5px;
```

Comment:
Please, use bootstrap for this padding.

Reply:
For this context is not possible to use bootstrap at this padding, it will works only with css